### DISCUSSION TITLE
DeskHop v0.68

### DISCUSSION THREAD

**Discussion by hrvach** [hrvach/deskhop#177] [Announcements]
DeskHop v0.68 (bugfixes)
  - Added screensaver shortcuts
  - Modified descriptor from 5+3 buttons to 8 buttons
  - Tidied up CMakeLists
  - Removed bootsel detection from non-debug version
  - Added OSHW cert

<hr /><em>This discussion was created from the release <a href='https://github.com/hrvach/deskhop/releases/tag/'>DeskHop v0.68</a>.</em>

**Comment by jalmeroth** [comment]
Hi @hrvach,

congrats on the OSHW cert very cool! :+1: 

Regarding the 8 button idea:
![swappy-20241125_222405](https://github.com/user-attachments/assets/db69a0b3-c3e0-42a2-83ca-60014ba10866)

I tested the new release with [facedancer](https://facedancer.readthedocs.io/en/latest/index.html) while recording the traffic. It looks, like the descriptor needs the `HID_USAGE_MAX` value set to 8 too. What do you think?

**Reply by hrvach** [reply]
Great catch, bumped to 8. Care to test again? :)

**Reply by jalmeroth** [reply]
That looks better now:
![swappy-20241125_223631](https://github.com/user-attachments/assets/623aba2e-65c5-46f7-a873-c023ac80fbc4)

**Reply by hrvach** [reply]
Really appreciate your inputs (no pun intended)!
